ui/src/App.tsx

```diff
@@ -26,8 +26,11 @@ function App(): JSX.Element {
             <Route exact path="/">
               <ShoeMap />
             </Route>
-            <Route path="/login">
-              <Login />
+            <Route exact path="/login">
+              <Login login={true} />
```

Comment:
it will have a value of **true** by default so you can just put **login** instead of explicitly defining